landoapi/api/revisions.py

```diff
@@ -26,6 +27,28 @@ def get(revision_id, api_key=None):
     return _format_revision(phab, revision, include_parents=True), 200
 
 
+def land(revision_id, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    phab = PhabricatorClient(api_key)
+    revision = phab.get_revision(id=revision_id)
+    revision = _format_revision(phab, revision, include_parents=True)
```

Comment:
Are the parent revisions needed here? Based on the google doc about the transplant service, we can only land 1 patch at a time anyway. Having the extra info doesn't hurt, but, keep in mind that getting the parents is an expensive operation as it could make upwards of 10+ calls to Phabricator for people that like to make chains of very small commits.